UIKitTask/Sources/Helpers/UIFont+Extension.swift

```diff
@@ -0,0 +1,19 @@
+// UIFont+Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Расширение для быстрого доступа к добавленным шрифтам
+extension UIFont {
```

Comment:
```suggestion
extension UIFont {
```
оформление не по гайду